src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,6 +28,31 @@
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private int getHash(K key) {
+        if (key == null) {
+            return 0;
+        }
+        return Math.abs(key.hashCode() % table.length);
+    }
+
+    private void resize() {
+        Node<K, V>[] currentTable = table;
+        table = (Node<K, V>[]) new Node[currentTable.length * RESIZE_NUMBER];
+
+    }
+
+    private static class Node<K, V> {
+        final K key;
+        final V value;
+        final Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
```

Comment:
Don't use getters and setters in class Node. We have access to private fields of inner class, so using getters or setters is redundant.